spec/support/webdrivers.rb

```diff
@@ -22,3 +22,5 @@
   ]
   WebMock.disable_net_connect!(allow_localhost: true, allow: allowed_sites)
 end
+
+Webdrivers::Chromedriver.required_version = '114.0.5735.90'
```

Comment:
I do not recommend setting the version explicitly unless this sets a minimum version. Chrome has a six-week release cycle, so we would need to change that bit often.

 In practice, devs need to regularly update their local Chromedriver binary as newer Chrome releases do not support older versions.

Reply:
Fixed in 7a85b3a20bdf4ace392c44f8fc65b3e521d4e3ce

Installed chromedriver version 115.0.5790.170 from https://googlechromelabs.github.io/chrome-for-testing/#stable